src/test/java/trackbot/ui/ParserTest.java

```diff
@@ -0,0 +1,86 @@
+package trackbot.ui;
+
+import trackbot.TrackBotException;
+import trackbot.commands.AddCommand;
+import trackbot.commands.Command;
+import trackbot.commands.DeleteCommand;
+import trackbot.commands.ExitCommand;
+import trackbot.commands.ListCommand;
+import trackbot.commands.MarkCommand;
+import trackbot.commands.UnmarkCommand;
+import trackbot.task.Deadline;
+import trackbot.task.Event;
+import trackbot.task.ToDo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ParserTest {
+    private final Parser parser = new Parser();
+
+    @Test
```

Comment:
Good use of a consistent format to write tests. They are very readable and easy to follow.